ember/app/templates/dependencies/index.hbs

```diff
@@ -0,0 +1,28 @@
+{{page-title "Dependencies"}}
+
+<div class="uk-child-width-1-2@s uk-flex uk-flex-column uk-flex-row@s gap@s">
+  <label for="name">Name
+    <Input
+      class="uk-input"
+      placeholder="Search by name..."
+      id="name"
+      @value={{this.name}}
+    />
+
+  </label>
+  <label for="provider">Provider
+    <PowerSelect
+      @onChange={{fn (mut this.provider)}}
+      @options={{this.providers}}
+      @placeholder="Select provider..."
+      @selected={{this.provider}}
+      @allowClear={{true}}
+      id="provider"
+      as |p|
+    >
+      {{p}}
+    </PowerSelect>
+
+  </label>
+</div>
```

Comment:
Could possibly become it's own component with some feature flags (status, provider, version, dependency). :man_shrugging:

Reply:
Done here: https://github.com/adfinis/Outdated/pull/170/commits/8ed9cef9776f2be5597fdea40db24c486e5b68b7

Reply:
Nice job! :+1: